### PR TITLE
Feature/entityfixes

### PR DESCRIPTION
**Pull request by shashi2k6**
Fixed entity relationship for the question and answer.

### CODE REVIEW

File: src/main/java/com/gal/trivia/controller/TriviaController.java [hunk 1]
```diff
@@ -15,7 +15,13 @@
     private TriviaService triviaService;
```
Comment (subhrajit-banerjee-official) [inline src/main/java/com/gal/trivia/controller/TriviaController.java:15]: Preferably use Constructor Injection in place of field Injection.

File: src/main/java/com/gal/trivia/entity/Answer.java [hunk 2]
```diff
@@ -1,28 +1,25 @@
 package com.gal.trivia.entity;
 
-import lombok.Data;
+import lombok.*;
 
 import javax.persistence.*;
 
 @Entity
-@Data
+@Getter
```
Comment (subhrajit-banerjee-official) [inline src/main/java/com/gal/trivia/entity/Answer.java:8]: In place of Getter, Setter AllArgsConstructor we can use @Data which supports all 3 together.

File: src/main/java/com/gal/trivia/entity/Question.java [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
 
 import javax.persistence.*;
 import java.util.Date;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
 @Entity
-@Data
+@Getter
```
Comment (subhrajit-banerjee-official) [inline src/main/java/com/gal/trivia/entity/Question.java:11]: In place of Getter, Setter AllArgsConstructor we can use @Data which supports all 3 together.

File: src/test/java/com/gal/trivia/TriviaApplicationTests.java [hunk 4]
```diff
@@ -1,34 +1,94 @@
...
+     * @throws Exception
      */
     @Test
     public void test_getRandom_Question() throws Exception {
-
         mockMvc.perform(get("/api/questions")).
-                andExpect(status().isOk())
-                .andExpect(jsonPath("$",hasSize(10)));
-
+                andExpect(status().isOk());
```
Comment (subhrajit-banerjee-official) [inline src/test/java/com/gal/trivia/TriviaApplicationTests.java:65]: Could be enhanced as per the test clause get random questions, currently testing only the status

File: src/main/java/com/gal/trivia/entity/Question.java (hunk 3 above)
Comment (shashi2k6) [reply src/main/java/com/gal/trivia/entity/Question.java:11]: It make sense.. Thanks will fix it

**Pull request merged**